src/pages/invite-preview.js

```diff
@@ -5,33 +5,11 @@ import Layout from '../components/layout'
 import cloudinary from '../utils/cloudinary'
 
 const InvitePreviewPage = () => {
-  const dummyImageUrl = cloudinary.url(
```

Comment:
The preview is shown within `invite.js`, so most likely this entire `invite-preview.js` file is not needed?